charts/deploy/templates/thanos-objstore-config-secret.yml

```diff
@@ -0,0 +1,20 @@
+{{- define "pelorus.thanos.objstore.config" }}
+type: s3
+config:
+  bucket: thanos
```

Comment:
we probably don't want to hardcode this, but allow (and instruct) the user to set a bucket name

Reply:
I've changed this to the default and provided some documentation on the variable I've added.